torch/testing/_internal/common_utils.py

```diff
@@ -646,7 +646,7 @@ def read_and_process():
             " To enable set the environment variable PYTORCH_RUN_DISABLED_TESTS=1")
 
 class TestCase(expecttest.TestCase):
-    precision = 1e-5
+    precision = 1e-7
```

Comment:
I'm curious what happens if we set this to `None`. I think it's only used when we also use some annotations or something -- ideally I'd like to remove this eventually to reduce the number of ways we specify precision.

Reply:
We could set it to None and it would just require checking for None when we compared it with the current atol later. It'd probably be easier to set it to zero. Previously this value was too high and would always override some of the default atol values, but 1e-7 is small enough that this doesn't happen anymore. 

Removing it would be nice but we'd have to provide another mechanism for generated tests to specify how they want to compare tensors.